GOVERNANCE.md

```diff
@@ -0,0 +1,163 @@
+
+### Lazy consensus
+
+Decision making typically involves the following steps:
+
+- Proposal
+- Discussion
+- Vote (if consensus is not reached through discussion)
+- Decision
+
+Any community member can make a proposal for consideration by the community. To initiate a discussion about a new idea, they should create an issue or submit a PR implementing the idea to the issue tracker. This will prompt a review and, if necessary, a discussion of the idea. The goal of this review and discussion is to gain approval for the contribution. Since most people in the project community have a shared vision, there is often little discussion to reach consensus.
+
+In general, as long as nobody explicitly opposes a proposal or PR, it is recognized as having the support of the community. This is called lazy consensus - that is, those who have not stated their opinion explicitly have implicitly agreed to the proposal's implementation.
+
+Lazy consensus is a fundamental concept within the project. This process allows a large group of people to reach consensus efficiently as someone with no objections to a proposal need not spend time stating their position.
+
+For lazy consensus to be effective, it is necessary to allow at least 48 hours before assuming that there are no objections to the proposal. This requirement ensures that everyone is given enough time to read, digest, and respond to the proposal. This time period is chosen to be as inclusive as possible of all participants, regardless of their location and time commitments.
+
+### Voting
+
+Not all decisions can be made using lazy consensus. Issues such as those affecting the strategic direction or legal standing of the project must gain explicit approval in the form of a vote. Every member of the community is encouraged to express their opinions in all discussions and all votes. However, only project maintainers have binding votes for the purposes of decision making.
+
+## Roadmap Creation
+
+Our [roadmap](https://github.com/lyft/amundsen/blob/master/docs/roadmap.md) gives an overview of what we are currently working on and what we want to tackle next. This helps potential contributors understand your project's current status and where it's going next, as well as giving a chance to be part of the planning.
+
+In this section, we describe the process we follow to create it, using request for comments documents (RFCs).
+
+### RFCs Process
+
+Most of the issues we see can be handled with regular GitHub issues. However, some changes are "substantial", and we ask that these go through a design process and produce a consensus among the Amundsen community.
+
+The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the roadmap. The high-level process looks like this:
+
+1. Contributor creates an RFC draft in the repository
```

Comment:
Do you have any recommendation on tool to use for RFC draft? would it be on google doc or *.md file on amundsen repo or some other way?

Reply:
We'll create a dedicated RFC repository within the organization. We are looking at the process of [Ember](https://github.com/emberjs/rfcs) or [Rust](https://github.com/rust-lang/rfcs) as a guide.